providers/edge/src/airflow/providers/edge/plugins/templates/edge_worker_hosts.html

```diff
@@ -100,30 +101,48 @@ <h2>Edge Worker Hosts</h2>
                 {% endfor %}
               </ul>
             </td>
-            <td>
-              {%- if host.state in ["idle", "running"] -%}
-                <form action="../edgeworkerhosts/status/maintenance/{{ host.worker_name }}/on" method="POST">
+            {%- if host.state in ["idle", "running"] -%}
+              <form action="../edgeworkerhosts/status/maintenance/{{ host.worker_name }}/on" method="POST">
+                <td>
                   <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
                   <button type="submit" style="padding: 10px 20px; background-color: blue; color: white; border: none; border-radius: 5px;">
                       Enter
                   </button>
-                </form>
-              {%- elif host.state in ["maintenance pending", "maintenance mode", "maintenance request"] -%}
-                <form action="../edgeworkerhosts/status/maintenance/{{ host.worker_name }}/off" method="POST">
+                </td>
+                <td>
+                  <textarea  name="maintenance_comment" rows="3" maxlength="128" required>{{ host.maintenance_comment }}</textarea>
+                </td>
+              </form>
+            {%- elif host.state in ["maintenance pending", "maintenance mode", "maintenance request"] -%}
+              <form action="../edgeworkerhosts/status/maintenance/{{ host.worker_name }}/off" method="POST">
+                <td>
                   <input type="hidden" name="csrf_token" value="{{ csrf_token() }}"/>
                   <button type="submit" style="padding: 10px 20px; background-color: blue; color: white; border: none; border-radius: 5px;">
                       Exit
                   </button>
-                </form>
-              {%- elif host.state in ["offline", "unknown", "offline maintenance"] -%}
-                <form action="../edgeworkerhosts/status/maintenance/{{ host.worker_name }}/remove" method="POST">
+                </td>
+                <td>
+                  <textarea  name="maintenance_comment" rows="3" maxlength="128">{{ host.maintenance_comment }}</textarea>
```

Comment:
The textarea looks a bit mis-placed. Can the HTML be a bit nicer? That hurts my eyes a bit :-D
(Does not need to be perfect, needs to be re-worked for Airflow 3 soon anyway.)